contracts/governance/Governor.sol

```diff
@@ -91,7 +91,7 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
      */
     function supportsInterface(bytes4 interfaceId) public view virtual override(IERC165, ERC165) returns (bool) {
         return
-            interfaceId == type(IGovernor).interfaceId ||
+            interfaceId == type(IGovernor).interfaceId ^ IGovernor.getProposalId.selector ||
```

Comment:
The changes to `IGovernor.sol` cause the interfaceId to change. 

This leaves us with 3 options:
- don't change the `supportsInterface` logic, which means the old interfaceId is no longer advertised through ERC-165, and we have a new one instead. That is somehow breaking, and tests need to be updated
- don't change the `IGovernor` interface. We can have the new function in `Governor` without it also being in the interface.
- do an xor here to continue exposing the old interfaceId.

Reply:
@ernestognw is in favor of #1.

I don't have strong opinions

@frangio WDYT?

Reply:
For me the default choice is option 3, what we did during 4.x:

>do an xor here to continue exposing the old interfaceId.